libs/vue/src/builders/dev-server/schema.d.ts

```diff
@@ -12,4 +12,12 @@ export interface DevServerBuilderSchema extends JsonObject {
   skipPlugins?: string;
   browserTarget: string;
   watch: boolean;
+  publicPath?: string;
+  css: {
+    requireModuleExtension?: boolean;
+    extract?: boolean | object;
+    sourceMap?: boolean;
+    loaderOptions: object;
```

Comment:
Should this property be optional?

Reply:
No. Even without a default value specified in the schema it defaults to an empty object.